inspec/docker/Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM     centos:centos7
+
```

Comment:
Maybe switch to an ubuntu 18 base image?  I feel like we know ubuntu pretty well ...

Reply:
maybe, but it'll be taken care of on a different branch.

Reply:
maybe, but it'll be taken care of on a different branch.